### ISSUE TITLE
Double "Tomorrow" headers

### ISSUE THREAD

**Issue by dmfs** [schedjoules/android-event-discovery-sdk#23]
Yesterday evening (few minutes to midnight) I got this list view on my phone:

![screenshot_2016-12-07-23-36-14](https://cloud.githubusercontent.com/assets/1614987/21002966/e9f00a98-bd28-11e6-812f-df9f8248059a.png)

Not sure how it happened. I can't reproduce it.

**Comment by lemonboston** [comment]
I suspect that this might have been related to not proper handling of time zone of the day header earlier, so when #17 is closed (corresponding PR is merged), this is going to be fixed.

But we can keep this one open and add a specific test case for checking the screen with Espresso for "Today" and "Tomorrow" when the local time of the first event is let's say 23:40. It will make sense to do that after #20 and #21 are closed.

**Comment by lemonboston** [comment]
I think this can be closed now since the PR for #17 has been merged.
(A test case has also been added to #56 to check this)

**Comment by dmfs** [comment]
Ok, I'll raise another issue if it happens again.
